feature/danggn/src/main/java/com/mashup/feature/danggn/ranking/DanggnRankingContent.kt

```diff
@@ -106,52 +107,90 @@ fun DanggnRankingContent(
-                                .fillMaxSize()
-                                .padding(
-                                    start = 20.dp,
-                                    end = 20.dp,
-                                    top = 6.dp,
-                                    bottom = 6.dp
-                                )
-                                .height(1.dp)
-                        ) {
-                            drawLine(
-                                color = Gray200,
-                                start = Offset(0f, 0f),
-                                end = Offset(size.width, 0f),
-                                pathEffect = pathEffect
-                            )
-                        }
-                    }
+@Composable
+private fun PagerContents(list: List<DanggnMemberRankResponse>) {
+    val listState = rememberLazyListState()
+    LazyColumn(
+        modifier = Modifier.fillMaxSize(),
+        state = listState,
+        contentPadding = PaddingValues(top = 12.dp)
+    ) {
+        itemsIndexed(
+            items = list.sortedByDescending {
+                it.totalShakeScore
+            },
+            key = { _, item ->
+                item.memberId
+            }) { index, item ->
+            RankingContent(
+                modifier = Modifier.fillMaxWidth(),
+                index = index,
+                name = item.memberName,
+                shakeCount = item.totalShakeScore,
+            )
+
+            if (index == 2) {
+                val pathEffect = PathEffect.dashPathEffect(floatArrayOf(10f, 10f))
+                Canvas(
+                    Modifier
+                        .fillMaxSize()
+                        .padding(
+                            start = 20.dp,
+                            end = 20.dp,
+                            top = 6.dp,
+                            bottom = 6.dp
+                        )
+                        .height(1.dp)
+                ) {
+                    drawLine(
+                        color = Gray200,
+                        start = Offset(0f, 0f),
+                        end = Offset(size.width, 0f),
+                        pathEffect = pathEffect
+                    )
                 }
             }
         }
+        // TODO index 11일때 가리는 것 추가해야됨 지금은 넣으면 안보이기 때문에 안넣음
+        item {
+            Text(
+                modifier = Modifier
+                    .padding(top = 28.dp)
+                    .fillMaxWidth(),
+                textAlign = TextAlign.Center,
+                text = "당근을 더 흔들어서 랭킹 안에 들어보세요",
+                style = Body3,
+                color = Gray500
+            )
+        }
+
+        item {
+            val coroutineScope = rememberCoroutineScope()
+            MashUpButton(
+                modifier = Modifier.padding(
+                    start = 20.dp,
+                    end = 20.dp,
+                    top = 28.dp,
+                    bottom = 20.dp
+                ),
```

Comment:
```suggestion
                modifier = Modifier.padding(horizontal = 20.dp).padding(top = 28.dp, bottom = 20.dp)
```
취향 차이일 수도 있겠지만 저는 이렇게  start와 end 패딩이 같으면 horizontal로 적용해요!
참고만 해주세용